src/datasets/batch_view/addons/reference.md

```diff
@@ -1,64 +1,27 @@
-# Addons Datasets
+# Addons
 
-<!-- toc -->
+Addon usage by client, partitioned by day.
 
-# Introduction
+This documentation is a work in progress. Work is
+being tracked in [bug 1364172](https://bugzilla.mozilla.org/show_bug.cgi?id=1364172)
 
-{{#include ./intro.md}}
+## Data Reference
 
-# Data Reference
-
-## Example Queries
-
-## Sampling
-
-It contains one or more records for every
-[Main Summary](../main_summary/reference.md)
-record that contains a non-null value for `client_id`.
+This dataset contains one or more records for every entry in the main ping table
+that contain a non-null value for `client_id`.
```

Comment:
```suggestion
that contains a non-null value for `client_id`.
```

Reply:
I think `contain` is actually the correct grammar here (yes, this is confusing): https://ell.stackexchange.com/a/72639